docs/user/openstack/install_upi.md

```diff
@@ -754,13 +799,31 @@ We will create three worker nodes here.
 
 The workers need no ignition override -- we can pass the unmodified `worker.ign` as their user data:
 
+### Compute Nodes ports
+
 ```sh
 for index in $(seq 0 2); do
-    openstack port create --network "$INFRA_ID-network" --security-group "$INFRA_ID-worker" --allowed-address ip-address=192.0.2.5 --allowed-address ip-address=192.0.2.6 --allowed-address ip-address=192.0.2.7 "$INFRA_ID-worker-port-$index"
-    openstack server create --image rhcos --flavor m1.large --user-data "worker.ign" --port "$INFRA_ID-worker-port-$index" "$INFRA_ID-worker-$index"
+    openstack port create --network "$INFRA_ID-network" --security-group "$INFRA_ID-worker" --allowed-address ip-address=192.0.2.5 --allowed-address ip-address=192.0.2.6 --allowed-address ip-address=192.0.2.7 --tag openshiftClusterID="$INFRA_ID" "$INFRA_ID-worker-port-$index"
 done
 ```
 
+### Compute Nodes Trunks (Required for Kuryr SDN)
+
+We will create the Trunks for Kuryr to plug the containers into the OpenStack SDN.
+
+```sh
+for index in $(seq 0 2); do
+    openstack network trunk create --parent-port "$INFRA_ID-worker-port-$index" "$INFRA_ID-worker-trunk-$index"
```

Comment:
Don't we need to tag trunk?

Reply:
For UPI is not needed as we're deleting based on the name of the resource. It would be good for consistency with IPI though.

Reply:
How is it being deleted in case of IPI?

Reply:
Sure, but then I don't understand why are we doing things differently in case of UPI?

Reply:
As we previously discussed, we do not list trunks on the CNO. So, we're safe.